secret_with_rotation/main.tf

```diff
@@ -0,0 +1,15 @@
+resource "aws_secretsmanager_secret" "secret_with_rotation" {
```

Comment:
Is this module used anywhere yet? Interested to see how it works out.

Reply:
It is being used now in terraform-analytics-common